test/can-view-parser-test.js

```diff
@@ -695,3 +696,42 @@ test('camelCase properties are encoded with on:, :to, :from, :bind bindings', fu
 
 	parser("<h1 on:aB='c' dE:to='f' gH:from='i' jK:bind='l'></h1>", makeChecks(tests));
 });
+
+testHelpers.dev.devOnlyTest('Warn on missing attribute value end quotes (canjs/can-view-parser#7)', function () {
+	var makeWarnChecks = function(input, texts) {
+		var count = 0;
+		var _warn = canDev.warn;
+		canDev.warn = function(text) {
+			equal(text, texts[count++]);
+		};
+
+		parser(input, {
+			start: function(tagName, unary) {},
+			end: function(tagName, unary) {},
+			attrStart: function(attrName) {},
+			attrEnd: function(attrName) {},
+			attrValue: function(val) {},
+			done: function() {}
+		});
+
+		equal(count, texts.length);
+
+		canDev.warn = _warn;
+	};
+
+	makeWarnChecks('<my-input {value}="name" (value)="updateNameOnEven(%viewModel.value)/>', [
```

Comment:
Sorry I wasn't clear. Can you use `testHelpers.dev.willWarn` also?

Reply:
No problem, I'll fix it :)